### PR TITLE
Reorganize testing tools

### PR DESCRIPTION
**Pull request by jimrollenhagen**
Remove the old and crusty Docker-based tests. They're out of date, slow,
and I don't believe anyone uses them.

Make run_tests.sh more flexible. Make it use the current test commands.
Provide a usage output, and options to run only unit tests or only
lint. Default to both.

Make Travis CI use run_tests.sh so we have a single common command used
by CI and devs to run tests.

Also update README to match.

### CODE REVIEW

**Comment by jimrollenhagen** [comment]
Issue #72

**Comment by ghost** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ After [installing Meteor](https://www.meteor.com/install), clone this repository
...
 Running tests
 ------------
-    npm install -g spacejam
-    
-    spacejam test-packages
+    # one-time setup
+    curl https://install.meteor.com | /bin/sh
+    npm install -g spacejam eslint
+    export PATH="$HOME/.meteor:$PATH"
+    cp settings.json.template setting.json
```
Comment (salas-wcd) [inline README.md:48]: Small typo:
should be 
`cp settings.json.template settingS.json`

Comment (ghost) [inline README.md:48]: @salas-wcd Feel free to create a PR to fix this.

Comment (thomwiggers) [inline README.md:48]: I've fixed this by directly committing to master -- this was such a trivial change I didn't feel the need for a PR.

**Pull request merged**